pkg/sql/drop_index.go

```diff
@@ -245,6 +245,14 @@ func (p *planner) dropIndexByName(
 		return nil
 	}
 
+	if idx.Unique && behavior != tree.DropCascade && constraintBehavior != ignoreIdxConstraint && !idx.CreatedExplicitly {
+		return errors.WithHint(
+			pgerror.Newf(pgcode.DependentObjectsStillExist,
+				"index %q is in use as unique constraint", idx.Name),
+			"use CASCADE if you really want to drop it.",
```

Comment:
hints should start with capital letter, i.e. `Use CASCADE if you really want to drop it.`